### PR TITLE
Add manual/scheduled query snapshot docs/ Closes #104.

### LINKED ISSUE #104: Add manual/scheduled query snapshot docs

### PR DESCRIPTION
**Pull request by MichaelBurgess**

### CODE REVIEW

File: docs/cloud/queries.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+<img src="/images/docs/cloud/cloud-query-schema-search.png" width="200pt"/>
+<br />
+
+Clicking the search result will automatically generate a query to select all the columns from that schema table, limited to 100 rows and will automatically run it for you.
+
+<img src="/images/docs/cloud/cloud-query-table-results.png" width="400pt"/>
+<br />
+
+If you click the `Edit` button you can amend the query, perhaps by selecting just the columns you're interested in, or adding a `where` clause to filter the results. Please note that we limit queries to 5,000 rows in the interactive query console.
```
Comment (johnsmyth) [inline docs/cloud/queries.md:28]: is it 5000 or 3000?   I expected it to be the query API limitation, which the docs say is 3000 https://redocly.github.io/redoc/?url=https://cloud.steampipe.io/api/latest/docs/openapi.json&nocors#tag/OrgWorkspaces/operation/orgWorkspace_postQuery.  Are the api docs wrong?

File: docs/cloud/queries.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+From any query snapshot, you can browse and navigate its snapshot history. We generate a hash of the query used in the snapshot, allowing us to find other snapshots relating to this query. 
+Click the **View** button at the top of the dashboard to display the menu.  You can select a date from the calendar and the snapshots from that date are listed beneath. Click any snapshot to open it.
+
+<img src="/images/docs/cloud/cloud-query-related-snapshots.png" width="200pt"/>
+<br />
+
+Clicking **All matching snapshots** at the bottom of the menu will take you to the snapshots page (you can also navigate to this page by clicking the **Snapshots** button at the top of the **Dashboards** pane).
+
+
+## Managing Snapshots
```
Comment (johnsmyth) [inline docs/cloud/queries.md:62]: Do you think we should include this entire section here?  or just link to https://steampipe-cp8jwleq3-turbot.vercel.app/docs/cloud/dashboards#managing-snapshots with a closing sentence from Browsing Snaphots?   It feels a little repeated here, but I dont feel strongly about it either...

Comment (MichaelBurgess) [reply docs/cloud/queries.md:62]: I had the same thought when writing it. I wasn’t strong either way, but perhaps linking across would be better?

File: docs/cloud/queries.md (hunk 1 above)
Comment (MichaelBurgess) [reply docs/cloud/queries.md:28]: Docs are wrong. It’s 5000. We’ll  update them 👍

File: docs/cloud/queries.md (hunk 2 above)
Comment (johnsmyth) [reply docs/cloud/queries.md:62]: I don't mind either way.  I'm going to approve, but if you want to try it the other way thats fine.

**Review by johnsmyth — APPROVED** [review]

**Pull request merged**